scripts/generator.py

```diff
@@ -63,7 +63,8 @@ def main():
 
 def argument_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--ref', action='store', help='git reference to use when building schemas')
+    parser.add_argument('--ref', action='store', help='Loads fields definitions from `./schemas` subdirectory from specified git reference. \
+                                                       Use with `--include experimental/schemas` to additionally load experimental fields from the target ref.')
```

Comment:
Tough thing to message in this cramped space :-) What about this?

```suggestion
                                                       Note that "--include experimental/schemas" will also respect this git ref.')
```